test/index.js

```diff
@@ -0,0 +1,57 @@
+const proxyquire = require('proxyquire').noCallThru()
```

Comment:
IIRC, Proxyquire merely adds proxied methods to a library, which means it doesn't totally stub out a bit of a module. NoCallThru specifies you don't want to actually run the function afterwards and merely stub it.